### DISCUSSION TITLE
interspeech presentation slides

### DISCUSSION THREAD

**Discussion by lubacien** [k2-fsa/sherpa#469] [General]
Hello,

At your interspeech workshop you said the slides would be uploaded here if i understood correctly, can you post them? 

Thank you,
Lucien

**Comment by csukuangfj** [comment]
@m-wiesner

Hi Matthew, is there a time plan to upload the slides?

**Comment by csukuangfj** [comment]
Please see
https://livejohnshopkins-my.sharepoint.com/:p:/g/personal/mwiesne2_jh_edu/EYqRDl8cIr5BsVDxi1MOW5EBUpdqh10WFkzqixPIFM63hg?e=u3lrmL
